docs/content/dagster-cloud/developing-testing/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,169 @@
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `locations.build.directory` | The directory in your project code that you want to deploy, the default is `.`. If you have sub-folders you can specific the path to only deploy that sub-project | string (path)            |
+| `locations.build.registry`  | The Docker registry to push your code location to in the registry                                                                                                 | string (docker registry) |
+
+Single code location:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: example_location
+    code_source:
+      package_name: my_dagster_project
+    build:
+      directory: ./
+      registry: 764506304434.dkr.ecr.us-west-2.amazonaws.com/branch-deployments-gh-action-test
+```
+
+Multiple code locations:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: hooli_data_eng
+    build:
+      directory: ./
+      registry: 764506304434.dkr.ecr.us-west-2.amazonaws.com/hooli-data-science-prod
+  - location_name: basics
+    code_source:
+      python_file: definitions.py
+    build:
+      directory: ./hooli_basics
+      registry: 764506304434.dkr.ecr.us-west-2.amazonaws.com/hooli-data-science-prod
+```
+
+## Python Executable
+
+By default, Dagster Cloud will use the python executable that is installed on the agent. If you want to use a different python executable, you can specify it in the dagster_cloud.yaml file. It might also be useful to have different python executables for different code locations.
+
+Parameter reference table:
+
+| Parameter                   | Description                            | Data type     |
+| --------------------------- | -------------------------------------- | ------------- |
+| `locations.executable_path` | The location of your python executable | string (path) |
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: hooli_data_eng
+    executable_path: venvs/path/to/dataengineering_spark_team/bin/python
+  - location_name: basics
+    code_source:
+      python_file: definitions.py
+    executable_path: venvs/path/to/ml_tensorflow/bin/python
+```
+
+## Other Configurations
+
+### Secrets Management for Serverless
+
+When using a serverless deployment, secrets can be configured in the Dagster UI. For more information, visit [Dagster Cloud environment variables](/dagster-cloud/developing-testing/environment-variables-and-secrets#dagster-cloud-environment-variables-and-secrets).
+
+For Dagster Cloud Hybrid deployments, making environment variables accessible is accomplished by adding them to your agent's configuration
```

Comment:
I'm a little confused what this section is doing here / how it relates to the dagster_cloud.yaml file